src/cogniteClient.ts

```diff
@@ -324,6 +328,29 @@ export default class CogniteClient {
     this.httpClient.setBaseUrl(baseUrl);
   };
 
+  /**
+   * Returns the base-url used for all requests.
+   */
+  public getBaseUrl(): string {
+    return this.httpClient.getBaseUrl();
+  }
+
+  /**
+   * Returns the default HTTP request headers, including e.g. authentication
+   * headers that is included in all requests. Headers provided per-requests is not
+   * included in this list. This function is useful when constructing API requests
+   * outside the SDK.
+   *
+   * ```js
+   * const customUrl = [...];
+   * const headers = client.getDefaultRequestHeaders();
+   * const result = await fetch(customUrl, { headers: new Headers(headers) });
```

Comment:
```suggestion
   * const result = await fetch(customUrl, { headers });
```
`new Headers(...)` can it work without a constructor? can you check plz? if so then we could simplify the example